microsoft/botframework-solutions
# [Virtual Assistant Sample][TypeScript] Update sample behavior

**[DEPENDS ON #1337]**

## Description
We updated the `sample-assistant` with the latest changes on #1337 for the `Virtual Assistant Template`.

Related to #1058 

## Testing Steps

1. Go to `AI\templates\Virtual-Assistant-Template\typescript\samples\sample-assistant`
1. Deploy a bot running `deploy.ps1` file inside of `deployment/scripts` folder
1. Run `npm install`
1. Run `npm run start`
1. Check that the assistant has the same behavior to C# version

## Checklist
